pages/index.vue

```diff
@@ -262,6 +262,14 @@
                       </button>
                     </label>
                     <input ref="payload" name="payload" type="file" @change="uploadPayload" />
+                    <button
+                        class="icon"
+                        @click="prettifyRequestBody()"
+                        v-tooltip="$t('prettify_body')"
+                        v-if="this.contentType.endsWith('json')"
```

Comment:
Only show Prettify button if `raw input` is enabled:

`v-if="rawInput && this.contentType.endsWith('json')"`

Reply:
Done :)